code/game/objects/items/weapons/manuals.dm

```diff
@@ -1522,10 +1522,11 @@
 /obj/item/book/manual/tcfl_pamphlet
 	name = "tau ceti foreign legion pamphlet"
 	desc = "A simple pamphlet containing information about the Tau Ceti Foreign Legion."
-	icon_state = "tcfl_book"
-	item_state = "book13"
+	icon_state = "tcfl_pamphlet"
```

Comment:
```suggestion
	icon_state = "tcfl_pamphlet"
	item_state = "paper"
```
Currently this will show a book in-hand. This would change it to show a paper in-hand. More fitting IMHO.

Code is fine otherwise.

Reply:
Oh right. I saw this but forgor to replace it.....it was 4 am Q_Q